test/DynamoCoreTests/PackageDependencyTests.cs

```diff
@@ -0,0 +1,299 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.IO;
+using Dynamo.Configuration;
+using Dynamo.Graph.Nodes;
+using Dynamo.Graph.Workspaces;
+using Dynamo.Interfaces;
+using Dynamo.Models;
+using Dynamo.PackageManager;
+using Dynamo.Scheduler;
+using Dynamo.Search.SearchElements;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using NUnit.Framework;
+
+namespace Dynamo.Tests
+{
+    internal class PackageDependencyTests : DynamoModelTestBase
+    {
+        public string PackagesDirectory { get { return Path.Combine(TestDirectory, "pkgs"); } }
+
+        protected override void GetLibrariesToPreload(List<string> libraries)
+        {
+            libraries.Add("ProtoGeometry.dll");
+            libraries.Add("DesignScriptBuiltin.dll");
+            libraries.Add("DSCoreNodes.dll");
+            base.GetLibrariesToPreload(libraries);
+        }
+
+        private PackageLoader GetPackageLoader()
```

Comment:
This function already exists in `PackageLoaderTests`: https://github.com/DynamoDS/Dynamo/blob/55482f600fdf4bbf5cf0ae4802df41570a22b04a/test/Libraries/PackageManagerTests/PackageLoaderTests.cs#L235. You could have moved this existing method to the common base class I suppose.